mysql/datadog_checks/mysql/mysql.py

```diff
@@ -399,6 +408,57 @@ def _collect_replication_metrics(self, db, results, above_560):
         results.update(self._get_replica_status(db, above_560, nonblocking))
         return REPLICA_VARS
 
+    def _collect_group_replica_metrics(self, db, results):
+        try:
+            with closing(db.cursor()) as cursor:
+                cursor.execute(SQL_GROUP_REPLICATION_MEMBER)
+                replica_results = cursor.fetchone()
+                status = self.CRITICAL
+                additional_tags = []
+                if replica_results is None:
+                    self.log.warning('Unable to get group replica status, setting it as CRITICAL')
+                else:
+                    status = self.OK if replica_results[1] == 'ONLINE' else self.CRITICAL
```

Comment:
What if `replica_results` has a length less than 3? We only check if it's not `None` right now.